src/utils/intervalUtils.ts

```diff
@@ -51,7 +51,7 @@ export const calculateQueryInterval = (
   }
 
   const intervalInMilliseconds = endTimestamp - startTimestamp;
-  const minuteIntervalLimitInMilliseconds = 3 * 60 * 60 * 1000; // MINUTE granularity for timeframes below 3h
+  const minuteIntervalLimitInMilliseconds = 3 * 60 * 60 * 1000 + 60000; // MINUTE granularity for timeframes below 3h 1m
```

Comment:
This should be done on another PR, I also need context what is this fix for ?

Reply:
Will create a seperate PR with explaining of context

Reply:
Here the PR: https://github.com/bitmovin/analytics-grafana-datasource/pull/96